[12:58:21] [Client thread/INFO]: Setting user: Player504
[12:58:21] [Client thread/INFO]: (Session ID is token:0:Player504)
[12:58:23] [Client thread/INFO]: LWJGL Version: 2.9.4
[12:58:23] [Client thread/INFO]: Reloading ResourceManager: Default
[12:58:23] [Sound Library Loader/INFO]: Starting up SoundSystem...
[12:58:23] [Thread-6/INFO]: Initializing LWJGL OpenAL
[12:58:23] [Thread-6/INFO]: (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
[12:58:24] [Thread-6/INFO]: OpenAL initialized.
[12:58:24] [Sound Library Loader/INFO]: Sound engine started
[12:58:26] [Client thread/INFO]: Created: 512x512 textures-atlas
[12:58:34] [Alt Login Thread/INFO]: Logging in with username & password
[12:58:38] [Client thread/INFO]: Connecting to mc.hypixel.net, 25565
[12:58:39] [Client thread/INFO]: [CHAT] A new §eOptiFine§f version is available: §eHD Ultra I7§f
[12:58:39] [Client thread/INFO]: [CHAT]                                      
[12:58:39] [Client thread/INFO]: [CHAT]                          
[12:58:40] [Client thread/INFO]: [CHAT] §b[MVP§9+§b] _Prof_HedgeHog_§f §6joined the lobby!
[12:58:40] [Client thread/INFO]: [CHAT] §aYou have §b3 §aunclaimed leveling rewards!
[12:58:40] [Client thread/INFO]: [CHAT] §eClick here to view them!
[12:58:42] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:
[12:58:43] [Client thread/INFO]: [CHAT]  §b>§c>§a>§r §6[MVP§2++§6] oe_gracie§f §6joined the lobby! §a<§c<§b<
[12:58:44] [Client thread/INFO]: [CHAT] §b[MVP§c+§b] Dudu_Guerreiro§f §6joined the lobby!
[12:58:44] [Client thread/INFO]: [CHAT]  §b>§c>§a>§r §6[MVP§0++§6] GigoBite§f §6joined the lobby! §a<§c<§b<
[12:58:47] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:
[12:58:54] [Client thread/INFO]: [CHAT] Sending you to mini690C!
[12:58:55] [Client thread/INFO]: [CHAT] §b[MVP§c+§b] ImNotCrab§f §6joined the lobby!
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeObjective(Scoreboard.java:218) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleScoreboardObjective(NetHandlerPlayClient.java:1903) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:64) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:10) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:55] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:58:56] [Client thread/INFO]: [CHAT]        
[12:58:56] [Client thread/INFO]: [CHAT] TheDutchEagle1 has joined (5/12)!
[12:58:56] [Client thread/INFO]: [CHAT] PolisEgil has joined (6/12)!
[12:58:56] [Client thread/INFO]: [CHAT] LaserSlime has joined (7/12)!
[12:58:56] [Client thread/INFO]: [CHAT] luisfsds has joined (8/12)!
[12:58:56] [Client thread/INFO]: [CHAT] papermash123 has joined (9/12)!
[12:58:56] [Client thread/INFO]: [CHAT] Natalooh has joined (10/12)!
[12:58:57] [Client thread/INFO]: [CHAT] The game starts in 12 seconds!
[12:58:57] [Client thread/INFO]: [CHAT] B1ACK_F3TH3R has joined (11/12)!
[12:58:57] [Client thread/INFO]: [CHAT] Chipyxl has joined (12/12)!
[12:59:00] [Client thread/INFO]: [CHAT] The game starts in 10 seconds!
[12:59:05] [Client thread/INFO]: [CHAT] The game starts in 5 seconds!
[12:59:06] [Client thread/INFO]: [CHAT] The game starts in 4 seconds!
[12:59:07] [Client thread/INFO]: [CHAT] The game starts in 3 seconds!
[12:59:08] [Client thread/INFO]: [CHAT] The game starts in 2 seconds!
[12:59:09] [Client thread/INFO]: [CHAT] The game starts in 1 second!
[12:59:11] [Client thread/INFO]: [CHAT] ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
[12:59:11] [Client thread/INFO]: [CHAT]                                   SkyWars
[12:59:11] [Client thread/INFO]: [CHAT] 
[12:59:11] [Client thread/INFO]: [CHAT]          Gather resources and equipment on your
[12:59:11] [Client thread/INFO]: [CHAT]      island in order to eliminate every other player.
[12:59:11] [Client thread/INFO]: [CHAT]         Go to the center island for special chests
[12:59:11] [Client thread/INFO]: [CHAT]                            with special items!
[12:59:11] [Client thread/INFO]: [CHAT] 
[12:59:11] [Client thread/INFO]: [CHAT] ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
[12:59:11] [Client thread/INFO]: [CHAT] Teaming is not allowed on Solo mode!
[12:59:11] [Client thread/INFO]: [CHAT] 
§a➲ §bLogan1003 §eand §b10 others §eactivated §c+3.0X coins for this game! Gain XP and coins by » CLICKING HERE! «
[12:59:11] [Client thread/INFO]: [CHAT] Buy Network Boosters at https://store.hypixel.net
[12:59:11] [Client thread/INFO]: [CHAT]  
[12:59:11] [Client thread/INFO]: [CHAT] Cages opened! FIGHT!
[12:59:30] [Client thread/INFO]: [CHAT] §7[3⋆] §a[VIP] TheDutchEagle1§f: oiops
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeObjective(Scoreboard.java:218) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleScoreboardObjective(NetHandlerPlayClient.java:1903) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:64) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:10) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeObjective(Scoreboard.java:218) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleScoreboardObjective(NetHandlerPlayClient.java:1903) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:64) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:10) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeObjective(Scoreboard.java:218) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleScoreboardObjective(NetHandlerPlayClient.java:1903) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:64) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:10) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:30] [Client thread/INFO]: [CHAT]                                      
[12:59:30] [Client thread/INFO]: [CHAT]                          
[12:59:30] [Client thread/INFO]: [CHAT] §aYou have §b3 §aunclaimed leveling rewards!
[12:59:30] [Client thread/INFO]: [CHAT] §eClick here to view them!
[12:59:33] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:
[12:59:40] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeObjective(Scoreboard.java:218) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleScoreboardObjective(NetHandlerPlayClient.java:1903) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:64) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:10) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:44] [Client thread/INFO]: [CHAT]                                      
[12:59:44] [Client thread/INFO]: [CHAT]                          
[12:59:44] [Client thread/INFO]: [CHAT] ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
[12:59:44] [Client thread/INFO]: [CHAT] 
[12:59:44] [Client thread/INFO]: [CHAT]                            Bed Wars Experience
[12:59:44] [Client thread/INFO]: [CHAT]           Level 0                                           Level 1
[12:59:44] [Client thread/INFO]: [CHAT]           [■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■]
[12:59:44] [Client thread/INFO]: [CHAT]                MN  LEVEL UP! You are now Level 1!  NM
[12:59:44] [Client thread/INFO]: [CHAT] 
[12:59:44] [Client thread/INFO]: [CHAT] You earned 500 Bed Wars Experience (First Join)
[12:59:44] [Client thread/INFO]: [CHAT] You earned 1x Bed Wars Loot Chest
[12:59:44] [Client thread/INFO]: [CHAT] 
[12:59:44] [Client thread/INFO]: [CHAT] ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
[12:59:44] [Client thread/INFO]: [CHAT] §aYou have §b3 §aunclaimed leveling rewards!
[12:59:44] [Client thread/INFO]: [CHAT] §eClick here to view them!
[12:59:50] [Client thread/INFO]: [CHAT] Sending you to mini620F!
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeObjective(Scoreboard.java:218) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleScoreboardObjective(NetHandlerPlayClient.java:1903) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:64) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:10) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1099) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:424) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:113) [bin/:?]
	at Start.main(Start.java:11) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:2012) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[12:59:51] [Client thread/INFO]: [CHAT]        
[12:59:51] [Client thread/INFO]: [CHAT] luisfsds has joined (11/16)!
[12:59:54] [Client thread/INFO]: [CHAT] pollito2570 has joined (12/16)!
[13:00:00] [Client thread/INFO]: [CHAT] pollito2570 has quit!
[13:00:03] [Client thread/INFO]: [CHAT] mamasita_mcc has joined (12/16)!
[13:00:08] [Client thread/INFO]: [CHAT] Wolfblitzer546 has joined (13/16)!
[13:00:13] [Client thread/INFO]: [CHAT] TheGost123 has joined (14/16)!
[13:00:14] [Client thread/INFO]: [CHAT] The game starts in 20 seconds!
[13:00:14] [Client thread/INFO]: [CHAT] We don't have enough players! Start cancelled.
[13:00:14] [Client thread/INFO]: [CHAT] TheGost123 has quit!
[13:00:14] [Client thread/INFO]: [CHAT] §7Wolfblitzer546§7: don't leave
[13:00:18] [Client thread/INFO]: [CHAT] CBDestroyz has joined (14/16)!
[13:00:19] [Client thread/INFO]: [CHAT] The game starts in 20 seconds!
[13:00:25] [Client thread/INFO]: [CHAT] luisfsds: brrr
[13:00:28] [Client thread/INFO]: [CHAT] RustyMikey480 has joined (15/16)!
[13:00:29] [Client thread/INFO]: [CHAT] The game starts in 10 seconds!
[13:00:31] [Client thread/INFO]: [CHAT] §7CBDestroyz§7: brrr
[13:00:34] [Client thread/INFO]: [CHAT] The game starts in 5 seconds!
[13:00:35] [Client thread/INFO]: [CHAT] The game starts in 4 seconds!
[13:00:36] [Client thread/INFO]: [CHAT] The game starts in 3 seconds!
[13:00:37] [Client thread/INFO]: [CHAT] The game starts in 2 seconds!
[13:00:38] [Client thread/INFO]: [CHAT] The game starts in 1 second!
[13:00:40] [Client thread/INFO]: [CHAT] ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
[13:00:40] [Client thread/INFO]: [CHAT]                                   Bed Wars
[13:00:40] [Client thread/INFO]: [CHAT] 
[13:00:40] [Client thread/INFO]: [CHAT]      Protect your bed and destroy the enemy beds.
[13:00:40] [Client thread/INFO]: [CHAT]       Upgrade yourself and your team by collecting
[13:00:40] [Client thread/INFO]: [CHAT]     Iron, Gold, Emerald and Diamond from generators
[13:00:40] [Client thread/INFO]: [CHAT]                   to access powerful upgrades.
[13:00:40] [Client thread/INFO]: [CHAT] 
[13:00:40] [Client thread/INFO]: [CHAT] ▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬▬
[13:00:44] [Client thread/ERROR]: Item entity 292 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:50] [Client thread/ERROR]: Item entity 357 has no item?!
[13:00:52] [Client thread/ERROR]: Item entity 377 has no item?!
[13:00:53] [Client thread/ERROR]: Item entity 387 has no item?!
[13:00:56] [Client thread/ERROR]: Item entity 578 has no item?!
[13:00:56] [Client thread/ERROR]: Item entity 578 has no item?!
[13:00:56] [Client thread/ERROR]: Item entity 578 has no item?!
[13:00:56] [Client thread/ERROR]: Item entity 578 has no item?!
[13:00:56] [Client thread/ERROR]: Item entity 578 has no item?!
[13:00:56] [Client thread/ERROR]: Item entity 578 has no item?!
[13:00:58] [Client thread/ERROR]: Item entity 626 has no item?!
[13:01:00] [Client thread/ERROR]: Item entity 662 has no item?!
[13:01:03] [Client thread/INFO]: Stopping!
[13:01:03] [Client thread/INFO]: SoundSystem shutting down...
[13:01:03] [Client thread/WARN]: Author: Paul Lamb, www.paulscode.com
